### PR TITLE
Updated setup.py

### PR DESCRIPTION
**Pull request by anirudh-hegde**
/kind feature

Enhances the app as I have added Kubernetes packages.

#### Does this PR introduce a user-facing change?
NA

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -60,9 +60,9 @@
     tests_require=TESTS_REQUIRES,
     extras_require=EXTRAS,
     packages=['kubernetes', 'kubernetes.client', 'kubernetes.config',
-              'kubernetes.watch', 'kubernetes.client.api',
-              'kubernetes.stream', 'kubernetes.client.models',
-              'kubernetes.utils', 'kubernetes.client.apis',
+              'kubernetes.watch', 'kubernetes.client.api', 'kubernetes.client.api_client'
+              'kubernetes.client.rest', 'kubernetes.stream', 'kubernetes.client.models',
+              'kubernetes_asyncio', 'kubernetes.utils', 'kubernetes.client.apis',
```
Comment (roycaihw) [inline setup.py:65]: It's unclear to me what issue this PR is addressing. Could you elaborate?

Comment (anirudh-hegde) [reply setup.py:65]: Aims to provide various functionalities necessary for interacting with Kubernetes clusters using the Kubernetes Python client libraries

Comment (roycaihw) [reply setup.py:65]: `kubernetes_asyncio` is not part of this repo. I think you can include it separately in your own environment?

Comment (anirudh-hegde) [reply setup.py:65]: > `kubernetes_asyncio` is not part of this repo. I think you can include it separately in your own environment?

However it's a python package which supports both core Kubernetes API resources and custom resource definitions (CRDs), making it a versatile tool for various Kubernetes-related tasks.

**Pull request closed without merging**